File: 32B - Borze.cc
#include <iostream>
#include <vector>
using namespace std;

int main() {
    string s;
    cin>>s;
    string ans="";

  for (int i = 0; i < s.length(); ++i) {
            if(s[i]=='.'){
              ans+='0';
            }
            else if(s[i]=='-' && s[i+1]=='.'){
              ans+='1';
              i++;
            }
            else if(s[i]=='-' && s[i+1]=='-'){
              ans+='2';
              i++;
            }
        }
        cout<<ans;
    return 0;
}

File: 32B - Borze.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.nextLine();
        StringBuilder ans = new StringBuilder();

        for (int i = 0; i < s.length(); ++i) {
            if (s.charAt(i) == '.') {
                ans.append('0');
            } else if (s.charAt(i) == '-' && s.charAt(i + 1) == '.') {
                ans.append('1');
                i++;
            } else if (s.charAt(i) == '-' && s.charAt(i + 1) == '-') {
                ans.append('2');
                i++;
            }
        }
        
        System.out.println(ans.toString());
        sc.close();
    }
}
